lib/retriable.rb

```diff
@@ -73,4 +69,26 @@ def retriable(opts = {})
       end
     end
   end
+
+  module Internal
+    def self.matched_exception?(on, exception)
```

Comment:
If I just make this method private on the Retriable module, we can't call it... This private Internal module was the only workaround I came up with... other ideas?

Reply:
Try `private_class_method :matched_exception?`.

Reply:
that was it!